frontend/src/main.tsx

```diff
@@ -10,11 +10,11 @@ import {
 import * as Sentry from '@sentry/react';
 
 // Globals has to be imported first, because it contains all the base CSS!
-// eslint-disable-next-line import/order
-import Globals from './Globals';
+
```

Comment:
is this previous comment irrelevant or should we keep globals first, and can j keep the eslint-disable? but maybe works regardless now

Reply:
It doesn't work; the preview's styles are literally broken 😅

Reply:
ya i hadn't looked since the changes to pass ci lmao, def need to revert this

Reply:
Idk how this happened I think it had something to do with me running eslint --fix at some point